packages/calypso-products/src/constants/features.ts

```diff
@@ -90,6 +90,7 @@ export const FEATURE_RECURRING_PAYMENTS = 'recurring-payments';
 export const FEATURE_PREMIUM_CONTENT_BLOCK = 'premium-content-block';
 export const FEATURE_PREMIUM_CONTENT_CONTAINER = 'premium-content/container';
 export const FEATURE_HOSTING = 'hosting';
+export const FEATURE_PREMIUM_DESIGN_TOOLS = 'premium-design-tools';
```

Comment:
Since this feature is meant to highlight Global Styles with a different marketing name, I think we should still use the existing feature rather than adding a new one:

https://github.com/Automattic/wp-calypso/blob/9a3f6d73f51f951b0cbba9247ff046b41b1ae029/packages/calypso-products/src/constants/features.ts#L269

Reply:
Sure thing.

Reply:
Done.